libc/src/sched/linux/sched_getcpucount.cpp

```diff
@@ -11,7 +11,7 @@
 #include "src/__support/common.h"
 #include "src/__support/macros/config.h"
 
-#include <sched.h>
+#include "hdr/types/cpu_set_t.h"
 #include <stddef.h>
```

Comment:
What does this file use from stddef.h?

Please explicitly include libc/hdr/types/size_t.h for size_t.